src/components/TodoList/TodoList.tsx

```diff
@@ -1,224 +1,111 @@
 /* eslint-disable */
 import React from 'react';
+import { Todo } from '../../types/Todo';
+import { useAppSelector, useAppDispatch } from '../../app/store';
+import { loadedTodos } from '../../features/todos';
+import { currentTodoSlice, isSelected } from '../../features/currentTodo';
+import { selectQuery, selectStatus } from '../../features/filter';
+import classNames from 'classnames';
+import { Filter } from '../../types/Filter';
```

Comment:
Ensure that the `Filter` type includes the values `ACTIVE`, `COMPLETED`, and `ALL` as used in the `filter` function. If these values do not match, it could lead to runtime errors.